Vagrantfile

```diff
@@ -0,0 +1,31 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+
+Vagrant.configure("2") do |config|
+
+  # Box Settings
+  config.vm.box = "ubuntu/trusty64"
+
+  # Provider Settings
+  config.vm.provider "v

```

Comment:
1. The line is broken
2. Do you know already that your applications needs those resources?